packages/kit/src/core/prerender/prerender.js

```diff
@@ -381,6 +407,17 @@ export async function prerender() {
 
 	await q.done();
 
+	// handle invalid fragment links
+	for (const [key, referrers] of expected_deeplinks) {
+		const [path, id] = key.split('#');
```

Comment:
this could break if there is another hash like `/foo#bar#baz`

Reply:
fixed